Tic-Tac-Toe/Game.cs

```diff
@@ -58,12 +60,18 @@ private void TakeTurns()
                 }
```

Comment:
If the user inputs an invalid input, it goes to line 59, right?

And then, you'll continue to determine the winner, even though the move is invalid?